ibis/backends/dask/executor.py

```diff
@@ -364,10 +364,23 @@ def visit(cls, op: ops.Sort, parent, keys):
         # 2. sort the dataframe using those columns
         # 3. drop the sort key columns
         ascending = [key.ascending for key in op.keys]
+        nulls_first = [key.nulls_first for key in op.keys]
+
+        if all(nulls_first):
+            na_position = "first"
+        elif not any(nulls_first):
+            na_position = "last"
+        else:
+            raise ValueError("dask does not support different columns ordering")
```

Comment:
We currently don't have any tests checking for ordering more than one column with different values of nulls_first. My guess is that we should add this kind of test in test_generic.

Reply:
Yep, that sounds right!